### DISCUSSION TITLE
How to install on ubuntu linux

### DISCUSSION THREAD

**Discussion by pancakers2001** [Wohlstand/TheXTech#30] [Q&A]
Sorry for the noob question, but how do you install this on Ubuntu 18.04, i've tried multiple methods but all has failed.

**Comment by Wohlstand** [comment]
Mainly it's a portable pack yet. Once you will build the thing, you'll get at your "output/" directory the set of binaries:
* `bin/thextech` executable
* `lib/*.so*` - a set of shared libaries (SDL2, SDL2_mixer_ext (aka MixerX), etc.) are was built in the place and linked to the executable above.
* To make game being useful, there are two ways:
  * make at your home directory the `~/.PGE_Project/thextech/` directory and put an assets pack into it: ([SMBX 1.3](https://wohlsoft.ru/TheXTech/_downloads/thextech-smbx13-assets-full.7z) or [Adventures of Demo](https://wohlsoft.ru/TheXTech/_downloads/thextech-adventure-of-demo-assets-full.7z) on your choice). It's "Debugging way".
  *  take any random folder, unpack assets into it, and put "thextech" executable and it's dependent SO libraries into it (mix assets and executable) to make the package. It's gaming way: you can compose different game packages and play them separately.

If you gonna to make re-distributible package to other machines, you'll need to use static build flags to use less external dependencies (see `.github/workflows/continuis_integration.yml`).
